src/include/config.h

```diff
@@ -1,6 +1,6 @@
 /*
  *	
- * Copyright (c) 2016-2018 Cisco Systems, Inc.
+ * Copyright (c) 2016-2019 Cisco Systems, Inc.
```

Comment:
I noticed this file uses struct configuration vs configuration_t typedef. Recommend using the typedef.

Reply:
done